Source/config/DBFuncs.js

```diff
@@ -0,0 +1,45 @@
+var MongoClient = require('mongodb').MongoClient;
+var url = "mongodb://localhost:27017/";
+var Building = require('../app/models/building-schema');
+var DataEntry = require('../app/models/data-entry-schema');
+var mongoose = require("mongoose");
+mongoose.createConnection("mongodb://localhost:27017/");
+var exports = module.exports = {};
+
+exports.addBuildingToDatabase = (entry) => {
```

Comment:
Why call a js file for this function rather than just do it in the route handler?